Parser/Device/HbbTv.php

```diff
@@ -51,6 +51,21 @@ public function parse(): ?array
         return $this->getResult();
     }
 
+    /**
+     * Returns if the parsed UA was identified as Shell Tv device
+     *
+     * @return bool
+     *
+     * @throws \Exception
+     */
+    public function isShellTv(): bool
```

Comment:
@sgiehl I added a new condition to this parser, or was it better to do it in mobiles.yml?

Reply:
guess it's fine that way. But we may need to release a new major version after merging to ensure the fork maintainers may have a closer look on the changes

Reply:
ok, give me a couple of days (2~) I'll add more tv brands to this PR